pkg/kubectl/cmd/convert/convert.go

```diff
@@ -61,6 +63,10 @@ var (
 		kubectl convert -f . | kubectl create -f -`))
 )
 
+func init() {
```

Comment:
This is not the correct place, you should move your change to https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/convert/import_known_versions.go and update it correctly to install these into `legacyscheme.Scheme`.